std_msgs/Int16 id
std_msgs/Byte objectClass
geometry_msgs/Point position